suite-native/accounts/src/components/AccountBalance.tsx

```diff
@@ -36,7 +40,11 @@ export const AccountBalance = ({ accountKey, accountName }: AccountBalanceProps)
                     <CryptoIcon size="large" name="btc" />
                 </Box>
                 <Box>
-                    <Text color="gray800">$ {account.formattedBalance}</Text>
+                    <Text color="gray800">
+                        {FiatAmountFormatter.format(account.formattedBalance, {
+                            currency: fiatCurrency.label,
```

Comment:
This is not good, we can't take care of selecting and passing fiat currency everywhere. We should add this to the context of providers. Can you coordinate with @bouchja1 he can give you more info how it works.